packages/explorer-2.0/components/StakingGuide/Step4.tsx

```diff
@@ -83,9 +83,7 @@ const Step4 = ({ goTo, nextStep }) => {
         ETH Balance:{' '}
         <span sx={{ fontWeight: 'bold' }}>
           {dataMyAccount?.account &&
-            parseFloat(Utils.fromWei(dataMyAccount.account.ethBalance)).toFixed(
-              2,
-            )}
+            parseFloat(dataMyAccount.account.ethBalance).toFixed(2)}
```

Comment:
What do you think of using `abbreviateNumber` here as well and getting rid of the `AND` statement since we can use null propagation operator ? 

```suggestion
abbreviateNumber(+dataMyAccount?.account?.ethBalance, 2)
```